google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseInfo.java

```diff
@@ -63,11 +62,39 @@ public Builder setDialect(Dialect dialect) {
       throw new UnsupportedOperationException("Unimplemented");
     }
 
+    /**
+     * Optional for creating a new database.
+     *
+     * <p>The proto descriptors input as byte[] to be used for the database.
```

Comment:
nit: Please use javadoc tags while writing javadoc. Use @param for the parameters description.)

Reply:
Done. Can you check it?